### ISSUE TITLE
MathJax Support

### ISSUE THREAD

**Issue by rain1024** [showdownjs/showdown#134]
Anybody here have ever used showdown with MathJax (for rendering latex) in angular?

**Comment by GlassGhost** [comment]
I'm trying to do something similar I found https://gist.github.com/xentec/6588519 but haven't got it working yet

**Comment by tivie** [comment]
@rain1024 @GlassGhost is right. You would need a custom extension for that.

**Comment by rain1024** [comment]
@tivie How can I do it?

Currently, I walk though this problem by used these tools separately. I use `setInterval` to update mathjax outside of angular. It's acceptable but not perfect.

I hope someone will make an extension for that soon.

**Comment by pdeschen** [comment]
@rain1024 you may follow the cookbook about writing extensions

https://github.com/showdownjs/showdown/wiki/extensions

Mathjax would be a nice addition!

**Comment by GlassGhost** [comment]
Hope this might help a lil bit, not my work but I did credit the author in the commit log.

Also, I'm still reading the wiki on the build process & how to add a test.

**Comment by GlassGhost** [comment]
BTW It's currently not working for me: @ http://minjslib.github.io/gh-template/ really would appreciate if this was added, and someone made a pull request showing me how to get it working, I did manage to get  the table plugin working.

**Comment by GlassGhost** [comment]
BUMP!

**Comment by pdeschen** [comment]
@GlassGhost I understand you're impatient of getting this working but bumping after less than one day is a bit rude. Anyhow, I've manage to get this working but it is clearly far from perfect:

http://jsfiddle.net/uauct8vy/16/

Some work is required to get this extension fully in.

**Comment by GlassGhost** [comment]
@pdeschen I'm sorry it's just my first pull request and I'm little excited.

Also I've been trying to get some more eyes and on my [gh-template](http://minjslib.github.io/gh-template/) project. I thought more than 0 people would be interested in this sort of thing.

**Comment by justathoughtor2** [comment]
I know it's significantly in the future from when this Enhancement issue was filed, but I did manage to get  a MathJax extension working with the showdown-ghost fork. It should work with regular showdown with a few modifications. I saved it as a simple Gist called mathjax.js, available (along with its README.md) at https://gist.github.com/justathoughtor2/7dbbecc912dfe4d3580b

Let me know what you all think!
